lib/utils.js

```diff
@@ -1,25 +1,102 @@
 'use strict';
 
 var qs = require('qs');
-var _ = require('lodash');
 
 var hasOwn = {}.hasOwnProperty;
 var toString = {}.toString;
 
 var utils = module.exports = {
 
+  // Note that in ES6 all uses of this can just be replaced with Object.assign.
+  // Given a few more years we should drop this function completely.
+  assign: function(o1, o2) {
+    for (var attr in o2) {
+      o1[attr] = o2[attr];
+    }
+    return o1;
+  },
```

Comment:
This is a pretty naive implementation. I'd feel much more comfortable bringing in something like the `object-assign` module to do just this. https://www.npmjs.com/package/object-assign